tasks/au/data.py

```diff
@@ -263,6 +216,65 @@ def _fetch_requirements(self):
         with (open('{}/{}'.format(dir_path, '{}_{}_requirements.json'.format(self.profile, self.year)))) as f:
             return json.load(f)
 
+    def _process_col(self, line, rows, session, column_reqs, cols, tablename, source, license, country, unittags, subsectiontags):
+        col_id = line[1]            # B: short
+        col_name = line[2]          # C: name
+        denominators = line[3]      # D: denominators
+        col_unit = line[5]          # F: unit
```

Comment:
I like doing multi-assignment for this kind of stuff:
`_, col_id, col_name, denominator, _, col_unit = line`
but it might complicate things (you have to match exactly the number of vatiables to the array length).

Ignorable, of course